mlflow/fastai/callback.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import os
+import shutil
+import tempfile
+from functools import partial
+import matplotlib.pyplot as plt
+
+import mlflow.tracking
+from mlflow.utils.autologging_utils import try_mlflow_log
+from mlflow.fastai import log_model
+
+from fastai.callback.core import Callback
+
+
+# Move outside, because it cannot be pickled. Besides, ExceptionSafeClass was giving some issues
```

Comment:
What happens if we use `ExceptionSafeClass`?

Reply:
I locally ran the tests with the code below and they all passed.

```suggestion
class __MLflowFastaiCallback(Callback, metaclass=ExceptionSafeClass):
    ...
```